### PR TITLE
Updated docs look and feel

### PR DESCRIPTION
**Pull request by brainJamStark**
Added the AxonCloud docs.
Re-arranged navigation layout to cater for look and feel changes.

### CODE REVIEW

File: docs/editions/intro.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* Dynamic responsive dashboards curated by experts
+* Adaptive Repair process that will never let you down
+* Efficient and intuitive backup you can rely on
+* Easy log-file association and interrogation
+* Reliable rolling restart through automation
+* Sophisticated alerting integration and routing
+* PDF reporting to quickly deliver insights across the team
+* Highly efficient protocol ensuring exceptional performance
+
+All of this is underpinned by an efficient bi-directional protocol ensuring exception performance and scale.
```
Comment (rgooding) [inline docs/editions/intro.md:14]: exception -> exceptional

File: docs/editions/enterprise_edition.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Scale-out with one-stop operations, full data retention, extended functionality and optional fully managed Cassandra.
+
+Book a consultation with a Cassandra Expert.
+
+<a class="book_expert" href="https://axonops.com/book-an-expert/">Book an Expert</a>
+
+## Enterprise edition features
+
+* Cloud or Self-Hosted, you choose.
```
Comment (rgooding) [inline docs/editions/enterprise_edition.md:13]: These limits are for the free version, this needs updating for the Enterprise edition

**Review by rgooding — CHANGES_REQUESTED** [review]

File: docs/editions/intro.md (hunk 1 above)
Comment (brainJamStark) [reply docs/editions/intro.md]: fixed

File: docs/editions/enterprise_edition.md (hunk 2 above)
Comment (brainJamStark) [reply docs/editions/enterprise_edition.md:13]: Removed and updated. Please check

**Review by rgooding — APPROVED** [review]

**Review by hshimizu — APPROVED** [review]

**Pull request merged**